### ISSUE TITLE
Feature Request: User Based Emoji Set

### ISSUE THREAD

**Issue by elmstedt** [gdpelican/retort#78]
Simple version: Allow full emoji list for Admins when using a limited emoji set.
More extensive version: Allow separate emoji sets by user group or trust level.

**Comment by angusmcleod** [comment]
@elmstedt Could you explain the use case / background for this a bit more?

**Comment by elmstedt** [comment]
Absolutely.

The tl;dr version is: I want to use a limited set of emojis as a way to allow voting on posts across multiple dimensions/factors/characteristics, and in my particular use case I want to be able to allow a subset of higher level users to vote on a different set of categories than three base users.

I fully understand this is not the intended use case for your plugin. And at some point, when/if I get this off the ground I'll likely get around to rolling my own, but for now this would be useful to me.

Now, to give you more context on the event you're interested.

I'm a teaching assistant for an R programming class at a university. I am cobbling together an online coding platform where my students would complete coding challenges using a different system (a Shiny web-app, not important or relevant to my request but I'm erring on the side of over-sharing).

After the submission window for each challenge is closed, my app will create a new topic for that challenge, along with a post for each group of similar solutions.

The idea them is that my students can then discuss each other's solutions and vote for the best one (using the **Question and Answer** plugin).

I intend to use your **retort** plugin to allow students to a very limited emoji set to tag solutions as having certain qualities,

🤓 = Clever
😇 = Best practice
😕 = Confusing or complicated

But, I'd also like the professor, TA's, and graders to be able to award other emoji like,

🥇 = Ideal solution
🤯 = Impressive
Etc.

Then the emoji counts for each solution would be used to, among other things, award special badges to the students.
